source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/tiled_camera_cfg.py

```diff
@@ -15,7 +15,13 @@
 
 @configclass
 class TiledCameraCfg(SensorBaseCfg):
-    """Configuration for a tiled rendering-based camera sensor."""
+    """Configuration for a tiled rendering-based camera sensor.
+
+    .. note::
+
+        Compared to other cameras, the clipping behavior cannot be altered as the backend implementation automatically
+        clips the values exceeding the maximum range to zero.
```

Comment:
This information seems a bit random here and not clear which clipping you refer to. Maybe explain properly in the class documentation instead of the config one.